### DISCUSSION TITLE
--recursive option

### DISCUSSION THREAD

**Discussion by protofolius** [comictagger/comictagger#526] [Q&A] [answered]
I am trying to start a run with recursive option , but it only handles the files in the current directory and never any of the files in any of the underlying directories .

-R, --recursive       Recursively include files in sub-folders.

I am calling it like this `ComicTagger -g -s -f --complicated-parser --overwrite -t cr -i -v -R "*.*"`

**Comment by protofolius** [comment, marked as answer]
Maybe useful for others to know that it needs to be called like this 

`ComicTagger -R -g -s -f --complicated-parser --overwrite -t cr -i -v "Y:\Path\*"`

**Reply by lordwelch** [reply]
Glad you figured it out. Yes, your first try only looks for files and directories that have a literal `.` in them in the current directory.
Also if you are doing recursive you don't need to use globbing you can just give it the directory you want:
`ComicTagger -R -s -f --complicated-parser --overwrite -t cr -i -v "Y:\Path\"`

**Reply by protofolius** [reply]
I also instead of `*.*` I tried `*` but that didnt work either . It needs the complete path before the wildcard in order for the command to work .
Once it works , it's awesome .
